src/store/modules/user.store.js

```diff
@@ -14,8 +15,40 @@ export const userStore = {
                     users => commit('getAllSuccess', users),
                     error => commit('getAllFailure', error)
                 );
-        }
+        },
+        setting({ dispatch, commit }, {
+            profilePicture,
+            username,
+            bio,
+            password1,
+            password2,
+        }) {
+            commit('settingRequest', { username });
+
+            userService.setting(profilePicture, username, bio, password1, password2)
+            .then(
+                user => {
+                    // console.log(user);
+                    let storedProfile = JSON.parse(localStorage.getItem('user'));
```

Comment:
This replacing (mutating) of user in localStorage needs to be done in `refreshUser` mutation.

Reply:
Where is the mutation function ? @vikas339

Reply:
Why is this still not moved into `refreshUser` mutation ?

Reply:
If already moved, what is this still doing here ?